### ISSUE TITLE
About the open-source license in Dorado

### ISSUE THREAD

**Issue by hasindu2008** [nanoporetech/dorado#142]
Hi Dorado team,

We really appreciate that ONT made this Dorado open-source basecaller, unlike close-sourced Guppy which created many barriers.
However, I tried to read this https://github.com/nanoporetech/dorado/blob/master/LICENCE.txt and realised it is far beyond my comprehension.  Can you please explain this "Oxford Nanopore Technologies PLC. Public License Version 1.0" in layman's terms? 

My specific question actually is, if we use some parts of Dorado code in a different project called A, can we license that different project A under the MIT license?

**Comment by Psy-Fer** [comment]
Hello,

Just to weigh in here. Looking at https://opensource.org/osd/ this licence seems to not follow point 1 and point 6

```
1. Free Redistribution
The license shall not restrict any party from selling or giving away the software as a component of an aggregate software distribution containing programs from several different sources. The license shall not require a royalty or other fee for such sale.

```

In the definition of "research only" restricting commercial advantages or monetary compensation. That restricts Free Redistribution


```
6. No Discrimination Against Fields of Endeavor
The license must not restrict anyone from making use of the program in a specific field of endeavor. For example, it may not restrict the program from being used in a business, or from being used for genetic research.
```

Restricting to "research only", and therefore commercial advantages or monetary compensation, it discriminates against being used in a business, for example, use in a clinical diagnostic test. 


By not meeting all the points of the open source definition, I would conclude that the "Oxford Nanopore Technologies PLC. Public License Version 1.0" is NOT open source.

Am I missing something here?

I guess this leads to the obvious question: _If I develop a clinical test that uses ONT and software under this licence and other software under other licences, can I provide access to that test for a fee, as any pathology provider would or is it restricted by this licence?_

**Comment by cjw85** [comment]
@Psy-Fer @hasindu2008,

Please contact support@nanoporetech.com regarding this inquiry.

**Comment by gringer** [comment]
My understanding is that it's an open source license, but not a free software license. In other words, people can look at the code and use it themselves, but there are restrictions on distribution, modification, and combination. That would be consistent with how ONT has worked with developers in the past.
